mysql-test/t/percona_heap_bug_ps296.test

```diff
@@ -0,0 +1,31 @@
+-- echo PS-296: inconsistent maximum key lenghts caused a crash in the query optimizer
+
+--disable_result_log
+create table a (id int unsigned not null auto_increment primary key);
```

Comment:
Please, also consider uppercase for SQL keywords - this will make the testcase more readable.

Reply:
Done